src/character.cpp

```diff
@@ -2684,7 +2690,7 @@ bool Character::practice( const skill_id &id, int amount, int cap, bool suppress
 {
     SkillLevel &level = get_skill_level_object( id );
     const Skill &skill = id.obj();
-    if( !level.can_train() || in_sleep_state() ||
+    if( !level.can_train() || in_sleep_state() || has_effect( effect_fearparalyze ) ||
```

Comment:
This prevents them from being trained, but that is not a fix. Characters which are paralyzed should not be *trying to dodge at all*.

Reply:
I don't know why I didn't think of that. I can put a check into melee.cpp for that stuff.